source/reference/methods.txt

```diff
@@ -876,3 +876,31 @@ Role Management Methods
    * - :method:`db.updateRole()`
 
      - Updates a user-defined role.
+
+Replication Methods 
+-------------------
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+
+     - Description
+
+   * - :method:`rs.freeze()`
+
+     - Makes the replica set member that ``mongosh`` is connected to 
+       ineligible to become primary for the specified duration.
```

Comment:
I know we are linking out to the manual for more info, but we could add a sentence here stating "The duration is specified in seconds." That way users have at least a baseline to run this method from this page.